TSOClient/tso.simantics/Model/VMObjectQueries.cs

```diff
@@ -180,6 +219,14 @@ public void RemoveObject(VMEntity obj)
                 if (list.Count == 0) ObjectsByGUID.Remove(guid);
             }
             RemoveCategory(obj, obj.GetValue(VMStackObjectVariable.Category));
+            if (obj.SemiGlobal != null)
+            {
+                if (obj.SemiGlobal.Iff.Filename != null)    //sanity check
```

Comment:
Too many spaces before `//sanity check`

Reply:
Solved in fc27d4be1bda305e6d4dc490743be9e8b6c21285